### ISSUE TITLE
Unable to programatically determine the outcome of a create/update taxonomy operation

### ISSUE THREAD

**Issue by anaoum** [redbrick-ai/redbrick-sdk#165]
Currently, when you run `project.create_taxonomy` or `project.update_taxonomy`, the success/failure of the operation is unknown because these methods do not return anything. Internally, they call `self.context.project.create_taxonomy` and `self.context.project.update_taxonomy`, which do return a bool. This bool should be passed on to the caller of the public API.

**Comment by pritamrungta** [comment]
The internal method bool is bit under defined (it'll always be true)
When the operation fails, create/update taxonomy raises an Exception

**Comment by pritamrungta** [comment]
Do you have instances where the method(s) didn't raise Exception, even though the operation wasn't successful?

**Comment by anaoum** [comment]
Yes, if I change an attribute type, i get the following message logged, but no exception raised:
```
[05:41:06] ERROR    Attribute type mismatch for attributeId 0                                                                                                                                                                                                                                                            
```

**Comment by pritamrungta** [comment]
Hi @anaoum
 
Thank you for bringing this issue to our notice.
We've released [v2.13.5](https://pypi.org/project/redbrick-sdk/2.13.5/), where these methods will raise ValueError when they fail.
Could you please give it a try and let us know if it works?

Thanks!

**Comment by anaoum** [comment]
v2.13.5 has resolved my issue. Thank you!
